container/secrets.go

```diff
@@ -81,5 +81,13 @@ func readFile(root, name string) ([]SecretData, error) {
 }
 
 func getHostSecretData() ([]SecretData, error) {
-	return readAll("/usr/share/rhel/secrets", "")
+	baseSecrets, err := readAll("/usr/share/rhel/secrets", "")
+	if err != nil {
+		return nil, err
+	}
+	overrideSecrets, err := readAll("/etc/container/rhel/secrets", "")
```

Comment:
This does not fail when the directory does not exist.
e.g if `/etc/container/rhel/secrets` does not exist, then it is just gonna return a slice with `baseSecrets` and vice versa.